src/libsync/owncloudpropagator.h

```diff
@@ -187,34 +188,42 @@ class OWNCLOUDSYNC_EXPORT PropagateDirectory : public PropagatorJob {
     // e.g: create the directory
     QScopedPointer<PropagateItemJob>_firstJob;
 
-    // all the sub files or sub directories.
-    QVector<PropagatorJob *> _subJobs;
+    // all the new and changed files without conflicts.
+    QLinkedList<PropagatorJob *> _syncJobs;
```

Comment:
It should be fine, I have choosen LinkedList to be able to easily put jobs both at the end and begining of the list. I need it for bundling and stuff. You know, everywhere there are pros and cons. Lets discuss it, but this is implementation detail